addons/mail/models/mail_thread.py

```diff
@@ -260,9 +260,10 @@ def create(self, vals_list):
 
         # subscribe uid unless asked not to
         if not self._context.get('mail_create_nosubscribe'):
+            default_followers = self.env['mail.followers']._add_default_followers(self._name, [], self.env.user.partner_id.ids, customer_ids=[])[0][0]
```

Comment:
There is a 'conflicting wip' where we are trying to make a technical model out of mail_follower.
https://github.com/odoo/odoo/compare/master...odoo-dev:master-improve-mail-followers-jgi

Since a user cannot create a follower anymore, we cannot write on message_follower_ids and need to create them by hand using insert_followers. I also wanted to remove the 'check_exiting' in this flow since we are creating the record there, it looked useless even when giving res_ids.

Both idea could be merged together by calling inset_followers with threads.ids outside of the loop.

The previous branch was stuck because of some augmented query count because of cache invalidation linked to operation order. I was waiting for new orm to check that but I think that @tde-banana-odoo  is working on it now. 

exept that, LGTM

Reply:
Well first to merge wins. As it is only code move I think it is valid.

Reply:
I'm not sure what is the policy with the query count but it seems below expectations in almost all tests now, so maybe there doesn't need to be a new test for this case, but just lower all existing values?

Reply:
Do a first commit that update query count, then update it in your optimization commit so that we can see the changes.

Reply:
I've reduced the number of queries to the minimum possible on local, I'll see if it works on runbot and readapt them higher if necessary.

Then I tested the current PR but both commit keep the same number of queries, they are just saving CPU time.